usagov/vote-gov-drupal
# Feature/vote 1244/migrate state page json schema

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1244

## Description (optional)
Migrate the JSON scheme for each registration type:
Online
Mail-in
In-person only
Not needed

## Deployment and testing (required, if applicable)

### Post-deploy steps
1. lando retune

### Testing steps
1. For each registration type:
a. View state page in browser
b. Inspect the script output using developer tools, and verify it matches the current site
(Note, the election day portion of the script has been removed)
